base_menu_visibility_restriction/demo/res_group.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="group_demo_user" model="res.groups">
+        <field name="name">Menu visibility restriction demo group</field>
+        <field eval="[(4, ref('base.user_demo'))]" name="users" />
+    </record>
+</odoo>
```

Comment:
I think this file is not needed due to the elimination of the ir_ui_menu.xml file.

Now the new group does nothing